src/julia_to_gap.jl

```diff
@@ -219,7 +219,7 @@ function julia_to_gap(
     elseif Wrappers.IsRecord(obj)
         ret_val = NewPrecord(0)
         recursion_dict[obj] = ret_val
-        for x in Vector{String}(Wrappers.RecNames(obj))
+        for x in Wrappers.RecNames(obj)::GapObj
```

Comment:
This was a genuine bug here -- we are missing code coverage here.

Reply:
Yes, this was wrong.
Looking at the function again, I am also not sure whether the statement
> We have to do something only if recursive conversion is required

is correct: Don't we have to make at least a shallow copy?